parlai/tasks/wizard_of_wikipedia/agents.py

```diff
@@ -31,11 +31,17 @@
 
 
 def _first_val(dictionary):
-    return list(dictionary.values())[0]
+    vals = list(dictionary.values())
```

Comment:
@stephenroller did you add that when you were helping me during the debugging?

Reply:
@emilydinan  did this in #1849.

Reply:
Then I should remove that from this commit.